node/resolvers/Mutations/Users.ts

```diff
@@ -11,6 +12,36 @@ import {
 
 const config: any = currentSchema('b2b_users')
 
+const setChangeSession = async (sessionParameters: any, countRetry = 0) => {
+  const retry = 5
+
+  const {
+    context: {
+      clients: { session },
+      vtex: { logger },
+    },
+    publicKey,
+    value,
+    sessionCookie,
+  } = sessionParameters
+
+  try {
+    await session.updateSession(publicKey, value, [], sessionCookie)
+  } catch (error) {
+    logger.error({
+      error,
+      message: 'setChangeSession.error',
```

Comment:
If we add a new parameter with the `countRetry` here, maybe we can remove the `logger.info` below?